synfig-core/src/tool/joblistprocessor.cpp

```diff
@@ -264,23 +264,41 @@ void process_job (Job& job)
 	else
 	{
 		VERBOSE_OUT(1) << _("Rendering...") << std::endl;
-		std::chrono::system_clock::time_point start_timepoint =
-            std::chrono::system_clock::now();
 
-		// Call the render member of the target
-		if(!job.target->render(&p))
-			throw (SynfigToolException(SYNFIGTOOL_RENDERFAILURE, _("Render Failure.")));
+		bool should_print_benchmarks = SynfigToolGeneralOptions::instance()->should_print_benchmarks();
+		double dur = 0.f;
+		int repeats = SynfigToolGeneralOptions::instance()->get_repeats();
+
+		for(int i = 0; i < repeats; i++)
+		{
+			std::chrono::system_clock::time_point start_timepoint =
+				std::chrono::system_clock::now();
+
+			// Call the render member of the target
+			if(!job.target->render(&p))
+				throw (SynfigToolException(SYNFIGTOOL_RENDERFAILURE, _("Render Failure.")));
+
+			if(should_print_benchmarks)
+			{
+				std::chrono::duration<double> duration =
+					std::chrono::system_clock::now() - start_timepoint;
 
-		if(SynfigToolGeneralOptions::instance()->should_print_benchmarks())
-        {
-            std::chrono::duration<double> duration =
-                std::chrono::system_clock::now() - start_timepoint;
+				dur += duration.count();
+			}
+		}
 
-            std::cout << job.filename.c_str()
-                      << _(": Rendered in ")
-                      << duration.count()
-                      << _(" seconds.") << std::endl;
-        }
+		if(should_print_benchmarks)
+		{
+			std::cout << job.filename.c_str()
+				<< _(": Rendered ")
+				<< repeats
+				<< _( " times in ")
+				<< dur
+				<< _(" seconds.")
```

Comment:
I suggest using milliseconds for benchmarks as it is a more appropriate unit of measurement in this case.
For example current output on test file:
```
./solid.sifz: Rendered 10 times in 0.00730508 seconds. Average time per render: 0.000730508.
```

Suggested:
```
./solid.sifz: Rendered 10 times in 7.30508 ms. Average time per render: 0.730508.
```

Also I notice repeated output:
```
synfig(13480) [09:54:40] info: 	tl: [-4,000000,2,250000], br: [4,000000,-2,250000], focus: [0,000000,0,000000]
./solid.sifz ==> ./solid.null: DONE
./solid.sifz ==> ./solid.null: DONE
./solid.sifz ==> ./solid.null: DONE
./solid.sifz ==> ./solid.null: DONE
./solid.sifz ==> ./solid.null: DONE
./solid.sifz ==> ./solid.null: DONE
./solid.sifz ==> ./solid.null: DONE
./solid.sifz ==> ./solid.null: DONE
./solid.sifz ==> ./solid.null: DONE
./solid.sifz ==> ./solid.null: DONE
./solid.sifz: Rendered 10 times in 8.02305 ms. Average time per render: 0.802305.
./synfig --repeats=10 --benchmarks --time=1 --target=null ./solid.sifz  0,04s user 0,03s system 97% cpu 0,076 total
```

It's better to fix this.

Reply:
Yes, I wasn't sure what to do with this. I guess I will try to just keep the render progress on the same one line for each render? Like it is for one render.

Reply:
![image](https://user-images.githubusercontent.com/33715613/224530952-2e618778-4795-4c8f-adae-df1a8405a956.png)

What should I do in case like this? @ice0

Reply:
Hi! Sorry for a late answer.
I suggest considering the entire benchmark as a single cycle. If this will introduce a lot of changes, when leave it as-is.

Reply:
Hi, sorry I dont understand what you mean by
> considering the entire benchmark as a single cycle

Like in the progress callback, I should modify it to account for number of repeats? Like,

```cpp
total = frames_count * repeats;
current_frame *= times_repeated; // times repeated is a local variable in RenderProgress
```

Reply:
Yes. But I am not sure about that decision.
I think clean output is preferable, but multiple output also makes sense.
So I suggest to merge this PR as is, and test that decision in another PR. 
What do you think?

Reply:
Yes that makes sense. I will make another PR for that then.